src/mds/Server.cc

```diff
@@ -1533,100 +1587,127 @@ void Server::dispatch_client_request(MDRequestRef& mdr)
   switch (req->get_op()) {
   case CEPH_MDS_OP_LOOKUPHASH:
   case CEPH_MDS_OP_LOOKUPINO:
+    logger->inc(l_mdss_req_lookupino);
     handle_client_lookup_ino(mdr, false, false);
     break;
   case CEPH_MDS_OP_LOOKUPPARENT:
+    logger->inc(l_mdss_req_lookupparent);
     handle_client_lookup_ino(mdr, true, false);
     break;
   case CEPH_MDS_OP_LOOKUPNAME:
+    logger->inc(l_mdss_req_lookupname);
     handle_client_lookup_ino(mdr, false, true);
     break;
 
     // inodes ops.
   case CEPH_MDS_OP_LOOKUP:
+    logger->inc(l_mdss_req_lookup);
     handle_client_getattr(mdr, true);
     break;
 
   case CEPH_MDS_OP_LOOKUPSNAP:
+    logger->inc(l_mdss_req_lookupsnap);
     // lookupsnap does not reference a CDentry; treat it as a getattr
   case CEPH_MDS_OP_GETATTR:
+    logger->inc(l_mdss_req_getattr);
     handle_client_getattr(mdr, false);
     break;
 
   case CEPH_MDS_OP_SETATTR:
+    logger->inc(l_mdss_req_setattr);
     handle_client_setattr(mdr);
     break;
   case CEPH_MDS_OP_SETLAYOUT:
+    logger->inc(l_mdss_req_setlayout);
     handle_client_setlayout(mdr);
     break;
   case CEPH_MDS_OP_SETDIRLAYOUT:
+    logger->inc(l_mdss_req_setdirlayout);
     handle_client_setdirlayout(mdr);
     break;
   case CEPH_MDS_OP_SETXATTR:
+    logger->inc(l_mdss_req_setxattr);
     handle_client_setxattr(mdr);
     break;
   case CEPH_MDS_OP_RMXATTR:
+    logger->inc(l_mdss_req_rmxattr);
     handle_client_removexattr(mdr);
     break;
 
   case CEPH_MDS_OP_READDIR:
+    logger->inc(l_mdss_req_readdir);
     handle_client_readdir(mdr);
     break;
 
   case CEPH_MDS_OP_SETFILELOCK:
+    logger->inc(l_mdss_req_setfilelock);
     handle_client_file_setlock(mdr);
     break;
 
   case CEPH_MDS_OP_GETFILELOCK:
+    logger->inc(l_mdss_req_getfilelock);
     handle_client_file_readlock(mdr);
     break;
 
     // funky.
   case CEPH_MDS_OP_CREATE:
+    logger->inc(l_mdss_req_create);
     if (mdr->has_completed)
       handle_client_open(mdr);  // already created.. just open
     else
       handle_client_openc(mdr);
     break;
 
   case CEPH_MDS_OP_OPEN:
+    logger->inc(l_mdss_req_open);
     handle_client_open(mdr);
     break;
 
     // namespace.
     // no prior locks.
   case CEPH_MDS_OP_MKNOD:
+    logger->inc(l_mdss_req_mknod);
     handle_client_mknod(mdr);
     break;
   case CEPH_MDS_OP_LINK:
+    logger->inc(l_mdss_req_link);
     handle_client_link(mdr);
     break;
   case CEPH_MDS_OP_UNLINK:
+    logger->inc(l_mdss_req_unlink);
   case CEPH_MDS_OP_RMDIR:
+    logger->inc(l_mdss_req_rmdir);
```

Comment:
`l_mdss_req_rmdir` gets incremented for unlink and rmdir (see fallthrough for unlink).